_posts/2020-07-29-community-update.md

```diff
@@ -0,0 +1,216 @@
+
+<hr>
+
+# Upcoming Events (and More!)
+
+We're happy to see the "high season" of virtual events approaching, with a lot of great conferences taking place in the coming month, as well as some meetups. Here, we highlight some of the Flink talks happening in those events, but we recommend checking out the complete event programs!
+
+As usual, we also leave you with some resources to read and explore.
+
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th>Category</th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><span class="glyphicon glyphicon glyphicon-console" aria-hidden="true"></span> Events</td>
+      <td><ul>
+        <b>Virtual Flink Meetup (Jul. 29)</b>
+        <p><a href="https://www.meetup.com/seattle-flink/events/271922632/">What’s new in Flink 1.11? + Q&A with Aljoscha Krettek</a></p>
+      </ul>
+      <ul>
+        <b>DC Thursday (Jul. 30)</b>
+        <p><a href="https://www.eventbrite.com/e/dc-thurs-apache-flink-w-stephan-ewen-tickets-112137488246?utm_campaign=Events%20%26%20Talks&utm_content=135006406&utm_medium=social&utm_source=twitter&hss_channel=tw-2581958070">Interview and Community Q&A with Stephan Ewen</a></p>
+      </ul>
+      <ul>
+        <b>KubeCon + CloudNativeCon Europe (Aug. 17-20)</b>
+        <p><a href="https://kccnceu20.sched.com/event/ZelA/stateful-serverless-and-the-elephant-in-the-room-stephan-ewen-ververica">Stateful Serverless and the Elephant in the Room</a></p>
+      </ul>
+      <ul>
+        <b>DataEngBytes (Aug. 20-21)</b>
+        <p><a href="https://dataengconf.com.au/">Change Data Capture with Flink SQL and Debezium</a></p>
+        <p><a href="https://dataengconf.com.au/">Sweet Streams are Made of These: Data Driven Development with Stream Processing</a></p>
+      </ul>
+      <ul>
+        <b>Beam Summit (Aug. 24-29)</b>
+        <p><a href="https://2020.beamsummit.org/sessions/streaming-fast-slow/">Streaming, Fast and Slow</a></p>
+        <p><a href="https://2020.beamsummit.org/sessions/building-stateful-streaming-pipelines/">Building Stateful Streaming Pipelines With Beam</a></p>
+      </ul>
+    </td>
+    </tr>
+    <tr>
+      <td><span class="glyphicon glyphicon-fire" aria-hidden="true"></span> Blogposts</td>
+        <td><ul>
+        <b>Flink 1.11 Series</b>
+        <li><a href="https://flink.apache.org/news/2020/07/14/application-mode.html">Application Deployment in Flink: Current State and the new Application Mode</a></li>
+        <li><a href="https://flink.apache.org/2020/07/23/catalogs.html">Sharing is caring - Catalogs in Flink SQL (Tutorial)</a></li>
+        <li><a href="">Flink SQL Demo: Building an End-to-End Streaming Application (Tutorial)</a></li>
```

Comment:
There was an issue with the first attempt at publishing the blogpost, so adding a placeholder to add the link at some point.